cms/djangoapps/contentstore/management/commands/force_publish.py

```diff
@@ -0,0 +1,70 @@
+"""
+Script for force publishing a course
+"""
+from django.core.management.base import BaseCommand, CommandError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from .prompt import query_yes_no
+from .utils import get_course_versions
+
+# To run from command line: ./manage.py cms force_publish course-v1:org+course+run
+
+
+class Command(BaseCommand):
+    """Force publish a course"""
+    help = '''
+    Force publish a course. Takes one arguments:
+    <course_id>: the course id of the course you want to publish forcefully
+    '''
+
+    def handle(self, *args, **options):
+        """Execute the command"""
+        if len(args) not in {1, 2}:
+            raise CommandError("force_publish requires 1 or more argument: <course_id> |commit|")
```

Comment:
In that case, by all means leave it as is. Better to keep the commands consistent.